フロントからのリクエストにCORSエラーが発生する

Dockerコンテナの起動時に以下のエラーが発生していたため、nilポインタを参照しないよう修正する

```bash
digicre@163-44-250-12:~/digicore_v3_backend$ docker logs 74dffa37f4c9
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xa9ba9a]

goroutine 1 [running]:
github.com/SIT-DigiCre/digicore_v3_backend/pkg/db.init.0()
        /work/pkg/db/db.go:31 +0x25a
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xa9ba9a]

goroutine 1 [running]:
github.com/SIT-DigiCre/digicore_v3_backend/pkg/db.init.0()
        /work/pkg/db/db.go:31 +0x25a
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xa9ba9a]

goroutine 1 [running]:
github.com/SIT-DigiCre/digicore_v3_backend/pkg/db.init.0()
        /work/pkg/db/db.go:31 +0x25a
```

--- a/pkg/db/db.go
+++ b/pkg/db/db.go
@@ -27,7 +27,9 @@ func init() {
 	if err != nil {
 		logrus.Fatal(err.Error())
 	}
-	if sdb.Ping() != nil {
+
+	err = sdb.Ping()
+	if err != nil {
 		logrus.Fatal(err.Error())
 	}
 	tw = twowaysql.New(sdb)

